<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamSettingClass</class>
 <widget class="QDialog" name="ParamSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParamSetting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>基准数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_line_base_file"/>
     </item>
     <item>
      <widget class="QPushButton" name="_button_base">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>5711数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_line_5711_file"/>
     </item>
     <item>
      <widget class="QPushButton" name="_button_5711">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>融合数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_line_merge_file"/>
     </item>
     <item>
      <widget class="QPushButton" name="_button_merge">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>纬度精度(度)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_lat_precise"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>经度精度(度)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_lon_precise"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_button_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_button_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
